lib/trento_web/controllers/v2/cluster_json.ex

```diff
@@ -75,4 +79,11 @@ defmodule TrentoWeb.V2.ClusterJSON do
   defp adapt_resources(resources) do
     Enum.map(resources, &Map.drop(&1, [:parent]))
   end
+
+  defp adapt_sids(%{sap_instances: sap_instances} = cluster) do
+    cluster
+    |> Map.put(:sid, SapInstance.get_hana_instance_sid(sap_instances))
+    |> Map.put(:additional_sids, SapInstance.get_sap_instance_sids(sap_instances))
+    |> Map.delete(:sap_instances)
```

Comment:
Removing the `sap_instances` by now.
I will work on the frontend part in a next implementation and see how to give the information to the frontend